### ISSUE TITLE
Used in non human species

### ISSUE THREAD

**Issue by guandailu** [davek44/Basset#56]
I am wondering if the software can be used in nonhuman species? If not, can we train a model for other species? Many thanks!

**Comment by davek44** [comment]
Yes, the software is generic for any species with a FASTA genome and some regulatory annotations that can be represented in a BigWig-like signal format.
